API-Server/src/main/java/swm/hkcc/LGTM/app/modules/auth/service/AuthServiceImpl.java

```diff
@@ -128,16 +133,14 @@ private void validateTagList(List<String> tagList) {
 
     private String createAccessToken(Member member) {
         return tokenProvider.createToken(
-                member.getMemberId(),
-                member.getGithubId(),
+                member.getNickName(),
```

Comment:
getNickName 을 githubId로 바꿔야 될 것 같습니다.
```suggestion
                member. getGithubId(),
```